### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by hover2pi**
Got Travis build working! Good riddance, astropy "helpers".

@exowanderer , I assigned this to you since you noticed the install wasn't working in the first place and made that other PR. Let me know if this succeeds. Thanks!

### CODE REVIEW

**Comment by julesfowler** [comment]
I see how it is. Wait to name the commit that fixes it `blame joe`.

File: ExoCTK/lightcurve_fitting/fitters.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 Email: jfilippazzo@stsci.edu
 """
 import numpy as np
-import lmfit
+# import lmfit
```
Comment (julesfowler) [inline ExoCTK/lightcurve_fitting/fitters.py]: Remove it or uncomment it pls.

Comment (hover2pi) [reply ExoCTK/lightcurve_fitting/fitters.py]: I'm scared.

Comment (julesfowler) [reply ExoCTK/lightcurve_fitting/fitters.py]: We all are.

**Comment by hover2pi** [comment]
@julesfowler Still passing!

**Comment by julesfowler** [comment]
#snailedit

**Pull request merged**